pkg/network-engine/gateway.go

```diff
@@ -34,6 +34,7 @@ import (
 
 const (
 	SecretFile string = "/etc/ipsec.d/raven.secrets"
+	DefaultPSK string = "alibaba-cloud-native"
```

Comment:
how about change `alibaba-cloud-native` to `openyurt-raven`

Reply:
done